apps/op-app/src/adapters/oidc/provider.ts

```diff
@@ -45,3 +51,6 @@ export const createProvider = (
       Session: 5 * 60,
     },
   });
+  provider.proxy = true;
```

Comment:
can you add a comment about why this is needed?

Reply:
done ✅